Containerfile

```diff
@@ -29,9 +29,9 @@ COPY --from=lightspeed-rag-content /rag/embeddings_model ./embeddings_model
 
 # Add explicit files and directories
 # (avoid accidental inclusion of local directories or env files or credentials)
-COPY runner.py requirements.txt ./
+COPY runner.py pyproject.toml LICENSE README.md ./
 
-RUN pip3.11 install --no-cache-dir -r requirements.txt
+RUN pip3.11 install --no-cache-dir .
```

Comment:
Since we had to run the `generate_packages_to_prefetch.py` script, I thought it would make more sense to go with https://github.com/road-core/service/pull/76 and simplify the process (and make it a bit faster)